share/package.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+if [ $# -eq 0 ]
+  then
+    echo "Path to package need to be specify"
+	exit 1
+fi
+
+if [[ `mkdir -p "work/gradle/wrapper"` -ne 0 ]];
+	then
+        echo "Cannot create work folder"
+        exit 1
+fi
+
+baseUrl=https://raw.githubusercontent.com/Cognifide/gradle-aem-plugin/master/share/wrapper
+declare -a arr=("gradlew" "build.gradle" "settings.gradle" "gradle/wrapper/gradle-wrapper.properties" "gradle/wrapper/gradle-wrapper.jar")
+
+for i in "${arr[@]}"
+do
+	response=$(curl "$baseUrl/$i" --write-out %{http_code} --silent --output "work/$i")  
```

Comment:
we are assuming that at least bash command is required to work with share mechanism

maybe validate command existence at the begging of the script and report nice error message when running from e.g cmd with no curl installed